OpenRA.Mods.Common/Traits/ThrowsShrapnel.cs

```diff
@@ -18,9 +18,15 @@ namespace OpenRA.Mods.Common.Traits
 	public class ThrowsShrapnelInfo : ITraitInfo
 	{
 		[WeaponReference, FieldLoader.Require]
-		public string[] Weapons = { };
-		public int[] Pieces = { 3, 10 };
-		public WDist[] Range = { WDist.FromCells(2), WDist.FromCells(5) };
+		[Desc("The weapons used for shrapnel.")]
+		public readonly string[] Weapons = { };
```

Comment:
Also, these documentation changes don't look directly related to spice blooms, so please split them into their own commit.